abb_irb4400_support/package.xml

```diff
@@ -3,18 +3,17 @@
   <version>0.1.0</version>
   <description>
     <p>
-      ROS Industrial support for the ABB IRB4400 (and variants).
+      ROS Industrial support for the ABB IRB4400/L30.
     </p>
     <p>
       This package contains configuration data, 3D models and launch files 
-      for ABB IRB4400 manipulators. This currently includes the L30.
+      for ABB IRB4400/L30 manipulator.
     </p>
     <p>
-      Joint limits and max joint velocities are based on the information in 
-      the ABB data sheets.  All URDFs / XACROs are based on the 
-      default motion and joint velocity limits, unless noted otherwise (ie: 
-      no support for high speed joints, extended / limited motion ranges or 
-      other options).
+      Joint limits and max joint velocities are based on the information in the <a href="http://www05.abb.com/global/scot/scot352.nsf/veritydisplay/c90b98aaa057bd6ec12576cb00528ef6/$file/Product%20specification%204400%20M99%20BWOS3.2.pdf">ABB 
+      IRB 4400 product specification document</a> (Article No: 3HAC 8770-1).  All URDFs / XACROs are based on the 
+      default motion and joint velocity limits, unless noted otherwise (ie: no support for high speed joints, 
+      extended / limited motion ranges or other options).
```

Comment:
It's almost impossible, but please try to keep this text within 80 columns. The URL is obviously too long, but it should be possible for the rest of the text.

Also, please lowercase "URDFs / XACROs".